backend/user/notifications.py

```diff
@@ -4,29 +4,41 @@
 from apns2.client import APNsClient
 from apns2.credentials import TokenCredentials
 from apns2.payload import Payload
+from celery import shared_task
 
 
 # taken from the apns2 method for batch notifications
 Notification = collections.namedtuple("Notification", ["token", "payload"])
 
 
+# celery delayed tasks
+def send_delayed_push_notif(token, title, body, countdown, isDev=False):
```

Comment:
nitpick here, I think in the `views.py` file you referenced `countdown` and `delay`. I would suggest using one or the other